docs/website/docs/command-reference/build-images.md

```diff
@@ -0,0 +1,28 @@
+---
+title: odo build-images
+sidebar_position: 1
+---
+
+# odo build-images
+
+odo can build container images based on Dockerfiles, and push these images to their registries.
+
+When running the command `odo build-images`, odo searches for all components in the `devfile.yaml` with the `image` type, for example:
+
+```
+components:
+- image:
+    imageName: quay.io/myusername/myimage
+    dockerfile:
+      uri: ./Dockerfile
+      buildContext: ${PROJECT_ROOT}
+  name: component-built-from-dockerfile
+```
+
+The `uri` field indicates the relative path of the Dockerfile to use, relative to the directory containing the `devfile.yaml`. The devfile specification indicates that `uri` could also be an HTTP URL, but this case is not supported by odo yet.
+
+The `buildContext` indicates the directory used as build context. The default value is `${PROJECT_ROOT}`.
```

Comment:
I think that this will have to be changed. I opened it here https://github.com/devfile/api/issues/668

Right now, we don't need to do anything about this, this is how it is in spec, so it is technically correct.